### ISSUE TITLE
Doesn't work with native iOS podcast app

### ISSUE THREAD

**Issue by scott113341** [L33T-KR3W/soundcasts-server#27]
Not sure why it doesn't work.  I'm guessing it's related to the SoundCloud streaming stuff, since it's a `206 Partial Content` response and not a normal file download:

```
HTTP/1.1 206 Partial Content
Content-Type: audio/mpeg
Content-Length: 76349439
Connection: keep-alive
Content-Range: bytes 0-76349438/76349439
```

**Comment by scott113341** [comment]
Fixed by #31

**Comment by scott113341** [comment]
Just to document what was happening:

Does not work: http://api.soundcasts.net/track/https%3A%2F%2Fapi.soundcloud.com%2Ftracks%2F254649516%2Fstream%3Fclient_id%3D83d37608cef935bb382915730917bb7d/track.mp3

Works: http://api.soundcasts.net/track.mp3?url=https%253A%252F%252Fapi.soundcloud.com%252Ftracks%252F254649516%252Fstream%253Fclient_id%253D83d37608cef935bb382915730917bb7d

I think some terrible URL parser was getting confused by the URL-encoded URL segment in the first example.  Moving the URL-encoded URL to a query parameter fixed the problem.
